src/eureka/S3_data_reduction/s3_reduce.py

```diff
@@ -349,6 +358,15 @@ def reduce(eventlabel, ecf_path=None, s2_meta=None):
                 if meta.verbose:
                     iterfn = tqdm(iterfn)
                 for n in iterfn:
+                    # when loop over ints, get exact y pos and width
+                    if meta.record_ypos:
+                        src_pos_results = source_pos.source_pos(data, meta,
+                                                                m, n,
+                                                                header=use_hdr)
+                        meta.src_ypos, ypos_exact, ypos_width = src_pos_results
```

Comment:
Instead of overwriting `meta.src_ypos` here which might have unintended consequences, use just a `_` instead to discard it

Reply:
done